cmd/main.go

```diff
@@ -31,19 +31,21 @@ func main() {
 
 	r := chi.NewRouter()
 
-	db, err := sql.Open("postgres", cfg.DBConnString)
+	ctxDb, cancel := context.WithTimeout(context.Background(), 10*time.Second)
+	defer cancel()
+	client, err := mongo.Connect(ctxDb, options.Client().ApplyURI("mongodb://localhost:27017"))
```

Comment:
Consider using configs instead of hardcoding db connection strings. Your solution won't work if you're working with several containers (ex. Docker Swarm)

Reply:
tnx for your feedback, i will fix it soon